src/app/lib/pageStyles/index.jsx

```diff
@@ -10,17 +11,22 @@ import {
   GEL_SPACING_TRPL,
   GEL_SPACING_QUAD,
   GEL_SPACING_QUIN,
-  GEL_MARGIN_BELOW_400PX,
-  GEL_MARGIN_ABOVE_400PX,
 } from '@bbc/gel-foundations/spacings';
-import styled from 'styled-components';
 
-const PageContainer = styled.div`
-  /* To add GEL Margins */
-  margin: 0 ${GEL_MARGIN_BELOW_400PX};
+export const PageMarginStyles = css`
+  margin: 0 ${GEL_SPACING};
   @media (min-width: ${GEL_GROUP_2_SCREEN_WIDTH_MIN}) {
-    margin: 0 ${GEL_MARGIN_ABOVE_400PX};
+    margin: 0 ${GEL_SPACING_DBL};
   }
+`;
+
+export const IndexMain = styled.main.attrs({ role: 'main' })`
```

Comment:
@sareh I've renamed it to `IndexMain`, but please note that this will only be used in the Most Read page. On the Front Page and IDX page, we add the margins to the `PageContainer` not to the `main`, since we want that some sections of the page have a background that takes the full width of the page like for example the `Ads` or `RadioSchedules`.